set.mm

```diff
@@ -20181,15 +20182,15 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
   $}
 
   ${
-    $d z w ph $.  $d x y ps $.  $d x y w $.  $d z y $.  $d x z $.
+    $d w z ph $.  $d x y ps $.  $d w x y z $.
     cbval2vw.1 $e |- ( ( x = z /\ y = w ) -> ( ph <-> ps ) ) $.
-    $( Version of ~ cbval2vv with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    $( Version of ~ cbval2vv with more disjoint variable conditions, which does
+       not require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
     cbval2vw $p |- ( A. x A. y ph <-> A. z A. w ps ) $=
       ( wal weq cbvaldvaw cbvalvw ) ADHBFHCECEIABDFGJK $.
 
-    $( Version of ~ cbvex2vv with a disjoint variable condition, which does not
-       require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
+    $( Version of ~ cbvex2vv with more disjoint variable conditions, which does
+       not require ~ ax-13 .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
     cbvex2vw $p |- ( E. x E. y ph <-> E. z E. w ps ) $=
       ( wex weq cbvexdvaw cbvexvw ) ADHBFHCECEIABDFGJK $.
```

Comment:
Here it's possible to remove ax-11 and ax-12 as well:
<pre>
  ${
    $d w z ph $.  $d x y ps $.  $d w x y z $.
    cbval2vw.1 $e |- ( ( x = z /\ y = w ) -> ( ph <-> ps ) ) $.
    $( Version of ~ cbval2vv with more disjoint variable conditions, which
       requires fewer axioms .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
    cbval2vw $p |- ( A. x A. y ph <-> A. z A. w ps ) $=
      ( wal weq wi wb biimp syl6com a2d cbvalivw 19.21v 3imtr3i pm2.86i equcomi
      ex biimpr syl56 com12 impbid cbvalvw ) ADHZBFHZCECEIZUFUGUHUFUGUHAJZDHZUH
      BJZFHZUHUFJZUHUGJZUIUKDFDFIZUHABUHUOABKZABJUHUOUPGTZABLMNOUHADPZUHBFPZQRU
      HUGUFULUJUNUMUKUIFDFDIZUHBAUHUTBAJZUTUOUHUPVAFDSUQABUAUBUCNOUSURQRUDUE $.

    $( Version of ~ cbvex2vv with more disjoint variable conditions, which
       requires fewer axioms .  (Contributed by Gino Giotto, 10-Jan-2024.) $)
    cbvex2vw $p |- ( E. x E. y ph <-> E. z E. w ps ) $=
      ( wex weq wn wal wi wb ex a2d cbvalivw 19.21v 3imtr3i pm2.86i alnex notbi
      syl6ib biimp syl6com equcomi biimpr syl56 impbid 3bitr3g con4bid cbvexvw
      com12 ) ADHZBFHZCECEIZUMUNUOAJZDKZBJZFKZUMJUNJUOUQUSUOUQUSUOUPLZDKZUOURLZ
      FKZUOUQLZUOUSLZUTVBDFDFIZUOUPURUOVFUPURMZUPURLUOVFABMZVGUOVFVHGNABUAUBZUP
      URUCUDOPUOUPDQZUOURFQZRSUOUSUQVCVAVEVDVBUTFDFDIZUOURUPUOVLURUPLZVLVFUOVGV
      MFDUEVIUPURUFUGULOPVKVJRSUHADTBFTUIUJUK $.
  $}
</pre>

I suspect that the same might be possible for `cbvaldvaw` and `cbvexdvaw` as well, but I don't have time to check it right now. If you feel like it, you can attempt to grab a piece (if not, I'll try some time in the future).